code/game/area/areas.dm

```diff
@@ -62,6 +62,9 @@
 	var/station_area = FALSE
 	var/centcomm_area = FALSE
 
+	var/narrate //A text-based description of what this area is for.
+	var/list/blurbed_stated_to = list() //This list of names is here to make sure we don't state our descriptive blurb to a person more than once.
```

Comment:
Document what this list is supposed to contain exactly. Names? As in names of mobs? Player mobs? Ckeys?